op-chain-ops/deployer/state/deploy_config.go

```diff
@@ -0,0 +1,154 @@
+package state
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+var (
+	l2GenesisBlockBaseFeePerGas = hexutil.Big(*(big.NewInt(1000000000)))
+
+	vaultMinWithdrawalAmount = mustHexBigFromHex("0x8ac7230489e80000")
+)
+
+func DefaultDeployConfig() genesis.DeployConfig {
+	return genesis.DeployConfig{
+		L2InitializationConfig: genesis.L2InitializationConfig{
+			L2GenesisBlockDeployConfig: genesis.L2GenesisBlockDeployConfig{
+				L2GenesisBlockGasLimit:      30_000_000,
```

Comment:
It might be nice to move these default values into variables instead of hardcoding. There are already some hardcoded default values [here](https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/genesis/genesis.go#L19-L41). It would be nice if both areas that use defaults pulled from a common place instead of hardcoding twice

Reply:
Agree with you there since one of the goals of `op-deployer` is to have one tool that handles deployments. I'd like to do that as a separate PR if possible, though - this one is getting pretty big and I suspect that'd be a larger refactor to coalesce all the defaults.